config/routes.rb

```diff
@@ -1,11 +1,19 @@
 Rails.application.routes.draw do
   root 'welcome#index'
+
   mount_devise_token_auth_for 'User', at: 'auth'
   namespace :api do
     namespace :v1 do
       resources :users
     end
   end
+
+  namespace :api do
```

Comment:
what is the idea to have the same namespace twice?

Reply:
Cause of my carelessness while resolving conflicts. Done.